api/src/pcapi/core/offers/api.py

```diff
@@ -24,13 +24,20 @@
 import pcapi.core.bookings.repository as bookings_repository
 from pcapi.core.categories import subcategories
 import pcapi.core.criteria.models as criteria_models
+from pcapi.core.educational import exceptions as educational_exceptions
 from pcapi.core.educational import models as educational_models
-from pcapi.core.educational.api import offer as educational_api_offer
+from pcapi.core.educational import validation as educational_validation
+from pcapi.core.educational.api import offer as educational_offers_api
 from pcapi.core.external.attributes.api import update_external_pro
 import pcapi.core.external_bookings.api as external_bookings_api
 import pcapi.core.finance.conf as finance_conf
 import pcapi.core.mails.transactional as transactional_mails
+from pcapi.core.offerers import api as offerers_api
+from pcapi.core.offerers import repository as offerers_repository
 from pcapi.core.offerers.models import Venue
+from pcapi.core.offers import offer_validation
```

Comment:
pourquoi changer cet import ?

Reply:
Je ne suis pas sur de comprendre mais l'ancien import était :
from . import offer_validation
Je me suis aligné sur la PEP8 : 
Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path). Mais hésite pas a me dire si c'est bien ca que tu souhaite que je remette